sdk/cognitiveservices/azopenai/autorest.md

```diff
@@ -286,7 +286,7 @@ directive:
     where: $
     transform: |
       return $
-        .replace(/\/\/ The model name.*?Model \*string/sg, "// DeploymentID specifies the name of the deployment (for Azure OpenAI) or model (for OpenAI) to use for this request.\nDeploymentID string");
+        .replace(/\/\/ The model name.*?Model \*string/sg, "// REQUIRED: DeploymentID specifies the name of the deployment (for Azure OpenAI) or model (for OpenAI) to use for this request.\nDeploymentID string");
```

Comment:
If you add this field to the `required` clause in the OpenAPI spec you'll get the comment for free plus it will group together with the other required fields.

Reply:
I'll add this for the next release (I had to generate the field "by hand" in autorest).

Reply:
Filed as #21234